### PR TITLE
Convert var -> let where can be done safely

### PR DESCRIPTION
**Pull request by benvinegar**
This was done w/ [jscodeshift](https://github.com/facebook/jscodeshift/pull/39/files) + [js-codemod/no-vars](https://github.com/cpojer/js-codemod/blob/master/transforms/no-vars.js) transform rules. I modified that script so that it only does `var` => `let` (removed `const` transformations) – mostly because it was declaring variables as `const` if they **happened** not to be reassigned in a block. I felt this didn't capture intent properly so I punted.

In cases where the codemod script blew up (didn't like functions declared in JSX prop passing), I converted manually.

Tests pass and the app seems to run fine. Might wanna throw on staging and do some passes to be extra sure.

**Update**: I made sure classes and function declarations are all `const`.

### CODE REVIEW

File: src/sentry/static/sentry/app/stores/groupStore.jsx [hunk 1]
```diff
@@ -4,14 +4,14 @@ import AlertActions from "../actions/alertActions";
...
 
-var ERR_CHANGE_ASSIGNEE = 'Unable to change assignee. Please try again.';
-var ERR_SCHEDULE_DELETE = 'Unable to delete events. Please try again.';
-var ERR_SCHEDULE_MERGE = 'Unable to merge events. Please try again.';
-var ERR_UPDATE = 'Unable to update events. Please try again.';
-var OK_SCHEDULE_DELETE = 'The selected events have been scheduled for deletion.';
-var OK_SCHEDULE_MERGE = 'The selected events have been scheduled for merge.';
-
-var GroupStore = Reflux.createStore({
+let ERR_CHANGE_ASSIGNEE = 'Unable to change assignee. Please try again.';
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/stores/groupStore.jsx]: See, these should be `const`.

File: src/sentry/static/sentry/app/stores/groupStore.jsx [hunk 2]
```diff
@@ -4,14 +4,14 @@ import AlertActions from "../actions/alertActions";
...
+let ERR_SCHEDULE_DELETE = 'Unable to delete events. Please try again.';
+let ERR_SCHEDULE_MERGE = 'Unable to merge events. Please try again.';
+let ERR_UPDATE = 'Unable to update events. Please try again.';
+let OK_SCHEDULE_DELETE = 'The selected events have been scheduled for deletion.';
+let OK_SCHEDULE_MERGE = 'The selected events have been scheduled for merge.';
+
+let GroupStore = Reflux.createStore({
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/stores/groupStore.jsx]: And probably also class declarations.

**Comment by dcramer** [comment]
Shouldn't classes be 'const'?

**Comment by benvinegar** [comment]
> Shouldn't classes be 'const'?

They are. Updated PR to reflect.

**Pull request merged**